versions/1.0.0.md

```diff
@@ -38,7 +42,10 @@ An overlay document contains an ordered list of [Action Objects](#overlayActions
 
 ### Versions
 
-TBD
+The Overlay Specification is versioned using a `major`.`minor`.`patch` versioning scheme. The `major`.`minor` portion of the version string (for example 1.0) SHALL designate the Arazzo feature set. `.patch` versions address errors in, or provide clarifications to, this document, not the feature set. The patch version SHOULD NOT be considered by tooling, making no distinction between 1.0.0 and 1.0.1 for example.
```

Comment:
```suggestion
The Overlay Specification is versioned using a `major`.`minor`.`patch` versioning scheme. The `major`.`minor` portion of the version string (for example 1.0) SHALL designate the Overlays feature set. `.patch` versions address errors in, or provide clarifications to, this document, not the feature set. The patch version SHOULD NOT be considered by tooling, making no distinction between 1.0.0 and 1.0.1 for example.
```

Which raises the question whether "Overlays" needs to to through a naming approval process, and whether it will come out as "Intonaco" or "Copertura" or something similar.

Reply:
Let's try to avoid that.